certbot/certbot/_internal/cli/__init__.py

```diff
@@ -182,7 +182,7 @@ def prepare_and_parse_args(plugins: plugins_disco.PluginsRegistry, args: List[st
         help=config_help("email"))
     helpful.add(["register", "update_account", "automation"], "--eff-email", action="store_true",
                 default=flag_default("eff_email"), dest="eff_email",
-                help="Share your e-mail address with EFF")
+                help="Share your e-mail address with EFF (default: False)")
     helpful.add(["register", "update_account", "automation"], "--no-eff-email",
                 action="store_false", default=flag_default("eff_email"), dest="eff_email",
                 help="Don't share your e-mail address with EFF")
```

Comment:
```suggestion
                help="Don't share your e-mail address with EFF (default: False)")
```

Reply:
Actually, the correct default here in both cases is `(default: Ask)` as we've used in other places.